modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/internal/content/processor/test/DefaultExportImportContentProcessorTest.java

```diff
@@ -869,6 +869,57 @@ public String getPathContext() {
 		portalUtil.setPortal(new PortalImpl());
 	}
 
+	@Test
+	public void testReplaceExportContentReferencesWithUUIDOneOnTrash()
```

Comment:
@AliciaGarciaGarcia, I would maybe call it  testReplaceExportContentReferencesWithDLFileEntryInTrash. No 100% sure though

Reply:
changed to `testReplaceExportContentReferencesWithFileEntryInTrash` (because is a FileEntry not a DLFIleEntry)

:D